src/re.rs

```diff
@@ -951,6 +951,15 @@ impl<'t> Captures<'t> {
 
 /// Get a group by index.
 ///
+/// `'t` is the lifetime of the matched text.
+///
+/// The text can't outlive the `Captures` object if this method is
+/// used, because of how `Index` is defined (normally `a[i]` is part
+/// of `a` and can't outlive it); to do that, use [`at()`][]
+/// instead.
+///
+/// [`at()`]: #method.at
```

Comment:
Could you remove the links here? I've generally avoided putting links to things in the docs because they're liable to become rotten.